StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/PaymentSheetVerticalViewController.swift

```diff
@@ -86,9 +86,9 @@ class PaymentSheetVerticalViewController: UIViewController, FlowControllerViewCo
     weak var paymentSheetDelegate: PaymentSheetViewControllerDelegate?
     let shouldShowApplePayInList: Bool
     let shouldShowLinkInList: Bool
-    /// Whether or not we are in the special case where we don't show the list and show the card form directly
-    var shouldDisplayCardFormOnly: Bool {
-        return paymentMethodTypes.count == 1 && paymentMethodTypes[0] == .stripe(.card)
+    /// Whether or not we are in the special case where we don't show the list and show the form directly
+    var shouldDisplayFormOnly: Bool {
```

Comment:
This isn't accurate, is it?

Reply:
Hrm, I guess not in the case there is no form to present when we don't collect user input. I've updated this flag to also do the checking for if the form requires user input.